chaco/colormapped_scatterplot.py

```diff
@@ -393,14 +390,10 @@ def _render_bruteforce(self, gc, points):
                 marker_size = marker_size[self._cached_point_mask]
             mode = marker_cls.draw_mode
 
-            if marker_cls != "custom":
+            if self.marker != "custom":
```

Comment:
marker_cls is not a string, but the actual class e.g. `<class 'enable.markers.CustomMarker'>`